app/src/main/java/com/example/recipeapp/auth/login/view/LoginFragment.kt

```diff
@@ -5,56 +5,40 @@ import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import android.widget.Button
+import android.widget.EditText
+import android.widget.TextView
+import androidx.navigation.fragment.findNavController
 import com.example.recipeapp.R
 
-// TODO: Rename parameter arguments, choose names that match
-// the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
-private const val ARG_PARAM1 = "param1"
-private const val ARG_PARAM2 = "param2"
-
-/**
- * A simple [Fragment] subclass.
- * Use the [LoginFragment.newInstance] factory method to
- * create an instance of this fragment.
- */
 class LoginFragment : Fragment() {
-    // TODO: Rename and change types of parameters
-    private var param1: String? = null
-    private var param2: String? = null
-
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-        arguments?.let {
-            param1 = it.getString(ARG_PARAM1)
-            param2 = it.getString(ARG_PARAM2)
-        }
-    }
+
+    private lateinit var email:EditText
+    private lateinit var password:EditText
+    private lateinit var loginButton: Button
+    private lateinit var registeTextView:TextView
 
     override fun onCreateView(
         inflater: LayoutInflater, container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
         // Inflate the layout for this fragment
-        return inflater.inflate(R.layout.fragment_login, container, false)
-    }
+        val view = inflater.inflate(R.layout.fragment_login, container, false)
+
+        email = view.findViewById(R.id.input_field_email)
+        password = view.findViewById(R.id.input_password)
+        loginButton = view.findViewById(R.id.button_login)
+        registeTextView = view.findViewById(R.id.go_to_sign_up)
 
-    companion object {
-        /**
-         * Use this factory method to create a new instance of
-         * this fragment using the provided parameters.
-         *
-         * @param param1 Parameter 1.
-         * @param param2 Parameter 2.
-         * @return A new instance of fragment LoginFragment.
-         */
-        // TODO: Rename and change types and number of parameters
-        @JvmStatic
-        fun newInstance(param1: String, param2: String) =
-            LoginFragment().apply {
-                arguments = Bundle().apply {
-                    putString(ARG_PARAM1, param1)
-                    putString(ARG_PARAM2, param2)
-                }
-            }
+
+        //fun login
+
```

Comment:
Use **dataBinding** instead of

`private lateinit var email:EditText
    private lateinit var password:EditText
    private lateinit var loginButton: Button
    private lateinit var registeTextView:TextView`
    
    ` email = view.findViewById(R.id.input_field_email)
        password = view.findViewById(R.id.input_password)
        loginButton = view.findViewById(R.id.button_login)
        registeTextView = view.findViewById(R.id.go_to_sign_up)`